platform/qemu.go

```diff
@@ -364,3 +378,89 @@ func Virtio(board, device, args string) string {
 	}
 	return fmt.Sprintf("virtio-%s-%s,%s", device, suffix, args)
 }
+
+func setupIgnition(confPath string, diskImagePath string) error {
+	fileRemoteLocation := "/boot/ignition/config.ign"
+
+	plog.Debugf("START guestfs")
```

Comment:
The two start/success `plog.Debugf` lines in this function don't feel particularly useful.

Reply:
@ashcrow removed the 2 `plog.Debugf`